src/Symfony/Bridge/Twig/NodeVisitor/TranslationNodeVisitor.php

```diff
@@ -158,6 +158,19 @@ private function getReadDomainFromNode(Node $node): ?string
             return $node->getAttribute('value');
         }
 
+        if (
+            $node instanceof FunctionExpression
+            && 'constant' === $node->getAttribute('name')
+        ) {
+            $nodeArguments = $node->getNode('arguments');
+            if ($nodeArguments->getIterator()->current() instanceof ConstantExpression) {
+                $value = \constant($nodeArguments->getIterator()->current()->getAttribute('value'));
```

Comment:
We should probably check that the constant is actually defined to prevent uncaught errors.

Reply:
I added `if (\defined($constantName)) {` ; do you think it's enough ?

Reply:
Will calling `defined()` trigger the autoloader in case a class constant is referenced?

Reply:
I dunno how the whole defined/constant works.

All I can say is that I tried this code in my project and all the missing translation are now found.

Reply:
I tried it, looks like it's taking into account.